website/docs/r/lb_nat_rule.html.markdown

```diff
@@ -80,9 +80,9 @@ The following arguments are supported:
 * `protocol` - (Required) The transport protocol for the external endpoint. Possible values are `Udp`, `Tcp` or `All`.
 * `frontend_port` - (Optional) The port for the external endpoint. Port numbers for each Rule must be unique within the Load Balancer. Possible values range between 1 and 65534, inclusive.
 * `backend_port` - (Required) The port used for internal connections on the endpoint. Possible values range between 1 and 65535, inclusive.
-* `frontend_port_start` - (Optional) The port range start for the external endpoint. This property is used together with BackendAddressPool and FrontendPortRangeEnd. Individual inbound NAT rule port mappings will be created for each backend address from BackendAddressPool. Acceptable values range from 1 to 65534, inclusive.
-* `frontend_port_end` - (Optional) The port range end for the external endpoint. This property is used together with BackendAddressPool and FrontendPortRangeStart. Individual inbound NAT rule port mappings will be created for each backend address from BackendAddressPool. Acceptable values range from 1 to 65534, inclusive.
-* `backend_address_pool_id` - (Optional) Specifies a reference to backendAddressPool resource.
+* `frontend_port_start` - (Optional) The port range start for the external endpoint. This property is used together with `frontend_port_end`. Individual inbound NAT rule port mappings will be created for each backend address from BackendAddressPool. Acceptable values range from 1 to 65534, inclusive.
+* `frontend_port_end` - (Optional) The port range end for the external endpoint. This property is used together with `frontend_port_start`. Individual inbound NAT rule port mappings will be created for each backend address from BackendAddressPool. Acceptable values range from 1 to 65534, inclusive.
+* `backend_address_pool_id` - (Optional) The ID of the Backend Address Pool resource.
```

Comment:
Can we mention the fact here that `backend_address_pool_id` has to be used together with the range, and otherwise, the range and the `frontend_port_id` are exclusively used?

Reply:
will leave the comment to let the user know